wgsl/index.bs

```diff
@@ -2300,6 +2300,10 @@ WGSL defines two <dfn>abstract numeric types</dfn> for these evaluations:
 An evaluation of an expression in one of these types [=shader-creation
 error|must not=] overflow or produce infinite, NaN, undefined, or [=indeterminate value|indeterminate=] results.
 
+Evaluation of [=AbstractFloat=] expressions may occur on the CPU implementing the WebGPU API methods.
```

Comment:
I think this is an implementation requirement is it not?

```suggestion
Evaluation of [=AbstractFloat=] expressions [=behavioral requirement|will=] occur on the CPU implementing the WebGPU API methods.
```

Reply:
It's what we expect to happen, but it could be done through a compile server on another agent.  I wanted to leave this loose.

Reply:
I see. I was thinking more along the lines of all abstracts are const-expressions and we say those will be evaluated at shader-creation time. This same comment could be said of all const-expressions could it not? Maybe it is better placed there (repeated in override-expressions possibly).

Reply:
Ok, I've moved it to the const-expression section and reworded it somewhat.  It still reads awkwardly because the reasoning isn't full prescriptive.   After all the accuracy for concrete float types are prescribed.
PTAL.

I didn't copy it to override-expression because override expressions always bottom out as concrete types. So the AbstractFloat discussion doesn't belong there, I don't think.